packages/testsuite/cypress.config.ts

```diff
@@ -29,48 +28,138 @@ export default defineConfig({
                 },
               ])
               .withWaitStrategy(Wait.forLogMessage(new RegExp(".*(WildFly Full.*|JBoss EAP.*)started in.*")))
-              .withStartupTimeout(333000)
-              .withCommand(["-c", configuration || "standalone-insecure.xml"] as string[])
+              .withStartupTimeout(30000);
+            if (useNetworkHostMode === true) {
+              console.log("host mode");
+              wildfly
+                .withNetworkMode("host")
+                .withCommand([
+                  "-c",
+                  configuration || "standalone-insecure.xml",
+                  `-Djboss.socket.binding.port-offset=${portOffset as string}`,
+                ] as string[]);
+            } else {
+              console.log("network mode");
```

Comment:
Network mode is `config.env.NETWORK_NAME` in this case. The log message does not make sense

Reply:
configured network added to log message